### ISSUE TITLE
GHC is missing a manpage

### ISSUE THREAD

**Issue by unode** [NixOS/nixpkgs#49627]
## Issue description
```
$ man ghc 
No manual entry for ghc
```

Several issues addressed this point by enabling sphinx documentation https://github.com/NixOS/nixpkgs/pull/21434 and https://github.com/NixOS/nixpkgs/issues/20281.
However as of `ghc-8.4.3` and `nixos-18.09` a manpage is missing from both outputs (`.doc` and `.out`).

In addition,  a manpage can still be found in other [distros](https://manpages.debian.org/unstable/ghc/ghc.1.en.html) so it seems something is preventing its creation.

## Technical details

```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.76, NixOS, 18.09.git.1bbd79f (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.1`
 - channels(unode): `"nixos-unstable-18.03pre117690.ab2cc75f78"`
 - nixpkgs: `/var/nixpkgs-channels/nixos`
```

**Comment by unode** [comment]
Pinging also https://github.com/NixOS/nixpkgs/pull/48806 in case this was addressed there.

**Comment by pbogdan** [comment]
Just noting that GHC 8.2 does include a man page and that version has sphinx as a build dependency. The dependency is not included for GHC 8.4 and 8.6. I wasn't able to find anything in git history indicating whether sphinx being omitted from newer versions is intentional or not.
